src/main/java/dinkplugin/notifiers/CombatTaskNotifier.java

```diff
@@ -100,6 +101,7 @@ private void handle(CombatAchievementTier tier, String task) {
             boolean crossedThreshold = prevThreshold > 0 && totalPoints - taskPoints < prevThreshold;
             CombatAchievementTier completedTier = crossedThreshold ? prev.getValue() : null;
             String completedTierName = completedTier != null ? completedTier.getDisplayName() : "N/A";
+            String currentTierName = crossedThreshold ? null : (prev != null ? String.valueOf(prev.getValue()) : null);
```

Comment:
```suggestion
            CombatAchievementTier currentTier = crossedThreshold || prev == null ? null : prev.getValue();
```

Reply:
Quick Java-noobie question here. Do we declare `currentTier`'s type with `CombatAchievementTier` since `CombatAchievementTier`  itself is a `enum`? If so, is it like saying` public enum currentTier = ...`